[2025-04-10T21:15:35.082+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: generate_data.run_generate_script scheduled__2023-01-17T00:00:00+00:00 [queued]>
[2025-04-10T21:15:35.111+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: generate_data.run_generate_script scheduled__2023-01-17T00:00:00+00:00 [queued]>
[2025-04-10T21:15:35.113+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-04-10T21:15:35.195+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): run_generate_script> on 2023-01-17 00:00:00+00:00
[2025-04-10T21:15:35.225+0000] {standard_task_runner.py:57} INFO - Started process 5110 to run task
[2025-04-10T21:15:35.245+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'generate_data', 'run_generate_script', 'scheduled__2023-01-17T00:00:00+00:00', '--job-id', '389', '--raw', '--subdir', 'DAGS_FOLDER/dags_1.py', '--cfg-path', '/tmp/tmp7nn3pdal']
[2025-04-10T21:15:35.251+0000] {standard_task_runner.py:85} INFO - Job 389: Subtask run_generate_script
[2025-04-10T21:15:35.383+0000] {task_command.py:415} INFO - Running <TaskInstance: generate_data.run_generate_script scheduled__2023-01-17T00:00:00+00:00 [running]> on host 7889a671eaba
[2025-04-10T21:15:35.804+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='generate_data' AIRFLOW_CTX_TASK_ID='run_generate_script' AIRFLOW_CTX_EXECUTION_DATE='2023-01-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-01-17T00:00:00+00:00'
[2025-04-10T21:15:38.696+0000] {logging_mixin.py:151} INFO - Script Output: 
[2025-04-10T21:15:38.697+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-10T21:15:38.712+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=generate_data, task_id=run_generate_script, execution_date=20230117T000000, start_date=20250410T211535, end_date=20250410T211538
[2025-04-10T21:15:38.749+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-10T21:15:38.777+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
